src/content/docs/how-to-work-on-quizzes.mdx

```diff
@@ -19,12 +19,20 @@ Every certification will have a series of quizzes to test camper's knowledge as
 
 ## Basic Structure for Quizzes
 
-Each quiz will have a total of 20 questions. Every quiz question will have a set of 4 options (1 correct answer and 3 distractors).
+Each quiz can have either a total of 10 or 20 questions. Every quiz question will have a set of 4 options (1 correct answer and 3 distractors).
 
-To pass the quiz, a camper will need to get at least 17 questions correct. If they fail the quiz, then there will be a brief cool down period before they are able to take the quiz again.
+To pass the quiz, a camper will need to get at least 90% of the questions correct. If they fail the quiz, then there will be a brief cool down period before they are able to take the quiz again.
 
 Before each quiz, there will be a review page listing out all of the topics that were previously discussed in the module.
 
+## Creating a new quiz
+
+In order to quickly create a new quiz run `pnpm run create-new-quiz` in a command prompt at the root of the project and it will add the boilerplate files for quizzes. This command runs a NPM script which prompts for relevant information for the quiz. 
```

Comment:
I don't think we need the second sentence. The first sentence covers it fine. Also, we want to make it clear that the core team will be the only ones creating the initial sets of new quizzes. 

```suggestion
The core team will be responsible for creating new quizzes for the certifications.

To create a new quiz, run `pnpm run create-new-quiz` in a command prompt at the root of the project. This will add the necessary boilerplate files for quizzes. 
```